### DISCUSSION TITLE
[答疑] 从零开始种植西红柿

### DISCUSSION THREAD

**Discussion by ZhuYuan-148** [share121/inter-knot#3347] [其他]
![mmexport1724163876624](https://github.com/user-attachments/assets/9181bd9f-4365-48d6-ac59-291af3d98c70)
首先，西红柿的种子一定要选取好，我会去光映广场的花店预定，花店老板提供的种子质量都很好。

其次，要适当浇水，旱季大概一天一次，雨季大概三天一次，还要保证充足的阳光

再次，大概长到15cm左右的西红柿苗比较脆弱，需要竹签支撑

最后，大概80天左右成熟，综合以上原因，最好种在自己经常待着的地方，可以有充足的时间来观察照顾，我就会选择在治安局，因为这里的阳光很足，我也可以在闲暇时光去照顾一下。只不过…在治安局也总是被偷吃…，青衣前辈说是这只猫吃的，但就算我再不喜欢毛茸茸，猫不吃西红柿这点我还是知道的吧，下次一定要抓住犯人狠狠的惩罚！

**Comment by HyperMuteki0347** [comment]
朱鸢长官，请问在西红柿结果期小果出现发紫发黑是正常现象吗？<img src="https://bbs-static.miyoushe.com/static/2023/11/16/b7b6eb7ebdc9f63ba6365e7e8d6b5fd7_5912345410510420967.png" alt="安比-啊···" width="75" >

**Reply by ZhuYuan-148** [reply]
emmm，大概是环境问题吧，我之前在室内种植也出现过，后来搬到治安局外面就好多了（大大增加了被偷吃的概率）

**Reply by HyperMuteki0347** [reply]
好的我一会移栽室外，等过几天再观察一下

**Comment by ShuiJingOfficial** [comment]
吃吃朱鸢的<img src="https://img-static.mihoyo.com/communityweb/upload/59353722c9fb0c5d926e84638b044e02.gif" alt="爱了爱了" width="75" >

**Reply by ZhuYuan-148** [reply]
等成熟了再来吃哦

**Comment by GrunderIndustries** [comment]
如果需要大批量种植作物或者压制犯人可以考虑我司的MQ-101
浇花战斗两不误

**Comment by OttoApocalypse01** [comment]
可以选择我们皮埃尔商店，最近在搞促销，作物种子半价哦

**Comment by zzztjs** [comment]
什么情况，治安官也来逛绳网了！？<img src="https://bbs-static.miyoushe.com/static/2023/11/16/b7b6eb7ebdc9f63ba6365e7e8d6b5fd7_5912345410510420967.png" alt="安比-啊···" width="75" >

**Reply by ZhuYuan-148** [reply]
其实前辈也来了哦。

**Comment by lip33625** [comment]
警官，我如果说，只是如果哦，如果真的吃了会怎样呢...我好像昨天...

**Reply by ZhuYuan-148** [reply]
先抓起来审问一天。
![Screenshot_20240820_170413_edit_65495136423339](https://github.com/user-attachments/assets/f108242e-7e23-49c8-b721-6014ee238e46)

**Reply by lip33625** [reply]
补药啊警官...不过有一说一，确实挺甜的

**Reply by ZhuYuan-148** [reply]
那…就算是帮助了市民吧，不抓了。

**Comment by jiangshixie** [comment]
朱鸢警官请逮捕我<img src="https://bbs-static.miyoushe.com/static/2024/07/31/4d5296bde00b3a911735a7599b9dc72e_3884466188049147.png" alt="彦卿 我来了" width="75" >

**Reply by ZhuYuan-148** [reply]
等我想个罪名再逮捕你

**Comment by Samu1KU** [comment]
感觉警官那边气候好好，我这雨季根本不用浇水，连下一个月叶子都给淹蔫巴了.......

**Reply by ZhuYuan-148** [reply]
新艾利都的气候真的很适合种农作物，外环那边太旱了，有的地方又太湿了，对番茄的种植都不太好。
